src/main/resources/alfresco/messages/system-messages.properties

```diff
@@ -114,3 +114,4 @@ system.license.err.reloadFailed=Failed to reload license: {0}
 # Startup message
 system.info.startup=Alfresco Content Services started ({0}{1}{2}). Current version: {3} schema {4}. Originally installed version: {5} schema {6}.
 system.warn.readonly=Alfresco Content Services is currently in read-only mode. Check your license.
+system.info.limited_trial=Note: this is a limited trial of the Enterprise version of Alfresco Content Services that goes into read-only mode after 2-days. Request an extended 30-day trial at: https://www.alfresco.com/platform/content-services-ecm/trial/docker
```

Comment:
Do you think it is worth taking the url out to somewhere else? If so, where would be a good place for it?